ote/src/cljs/ote/style/admin.cljs

```diff
@@ -7,4 +7,8 @@
                        :color         "#323232"
                        :font-weight   "400"
                        :text-align    "right"
-                       :padding-right "20px"})
\ No newline at end of file
+                       :padding-right "20px"})
+
+(def button-container {:display "flex"
```

Comment:
Just double-checking, is new button container style needed and following can't be reused?
`ote.ui.buttons.button-container`
It's used in most of other buttons and views.